### DISCUSSION TITLE
Cannot download GTFS data with free API access

### DISCUSSION THREAD

**Discussion by TheRabidWolverine** [transitland/transitland#418] [Q&A] [answered]
My account has a free API access with API key.

I am trying this basic download in web browser to download the latest GTSF file for an operator:

https://www.transit.land/api/v2/rest/feeds/f-c28-thevictoriaclipper/download_latest_feed_version?apikey=my_key

I keep getting an error, and no download:

{"error":"not authorized"}

In fact, even the base URL hit with API key returns the same error:

https://www.[transit.land/api/v2/feeds?api_key=my_key](https://www.transit.land/api/v2/feeds?api_key=my_key)

Any help on how to fix this?

**Comment by drewda** [comment, marked as answer]
@TheRabidWolverine please remove the `www` from the URL and try again.

For example: `https://transit.land/api/v2/rest/feeds/f-c28-thevictoriaclipper/download_latest_feed_version?apikey=my_key`

**Reply by TheRabidWolverine** [reply]
Wow. That works. Thanks. We are planning to upgrade to the pro plan. One problem - there doesn't seem to be a comprehensive list of all operators and routes present in the database. Going to the [base page](https://www.transit.land/operators?search=) and keeping on loading more is difficult, made even more so by the fact that new operators and datasets will keep getting added all the time. Is there any endpoint which can give us list of the operators?

**Reply by drewda** [reply]
See the REST API operator endpoint: https://www.transit.land/documentation/rest-api/agencies#operator-request-parameters

**Reply by TheRabidWolverine** [reply]
Perfect. Thanks a lot!
